src/sentry/replays/models.py

```diff
@@ -58,3 +58,9 @@ def delete(self, *args, **kwargs):
             file.delete()
 
         return rv
+
+
+# For the register decorator to work the code has to be executed. Since
```

Comment:
i believe you can remove this and add it to https://github.com/getsentry/sentry/blob/44886109dc69cd8aa21163606bd68bf70276de67/src/sentry/api/serializers/models/__init__.py#L1